src/pal/src/init/pal.cpp

```diff
@@ -436,7 +436,7 @@ Initialize(
         if (NO_ERROR != palError)
         {
             ERROR("Unable to create initial process and thread objects\n");
-            goto CLEANUP4;
+            goto CLEANUP2;
```

Comment:
Just curious, in this context is it possible to refactor the code so the (evil) `goto` statements are replaced by functions?